test/recipes/25-test_verify.t

```diff
@@ -350,6 +350,8 @@ SKIP: {
     ok(verify("ee-cert-ec-named-named", "", ["root-cert"],
               ["ca-cert-ec-named"]),
         "accept named curve leaf with named curve intermediate");
+    ok(verify("ee-cert-ec-sha3-384", "", ["root-cert"], ["ca-cert-ec-named"], ),
+        "accept cert generated with EC and SHA3-384");
```

Comment:
Could you please add this to the following section as well - this way the support is verified also with fips+base providers.